ml/try_real_photo.py

```
@@ -0,0 +1,225 @@
+            best = (fit, box)
+    if best is None:
+        raise SystemExit("no run could be read")
+    (score, count, pitch, start, confidence, predicted), box = best
+    print(f"\nchose {box}: {count} tiles, pitch {pitch:.1f}px, mean confidence {score:.3f}\n")
+    for i, (guess, sure) in enumerate(zip(predicted.tolist(), confidence.tolist()), 1):
+        mark = "" if sure >= CONFIDENT else "   <- hand this one back"
+        print(f"  {i:2d}. {labels[guess]:3s} {sure:.2f}{mark}")
+    kept = sum(1 for s in confidence.tolist() if s >= CONFIDENT)
+    print(f"\n{kept}/{count} at confidence >= {CONFIDENT}")
+
+    x, y, w, h = box
```

Ruff (0.16.1)

[warning] Unpacked variable `x` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)